tests/main/snapctl-is-connected-list/task.yaml

```
@@ -0,0 +1,16 @@
+summary: Ensure that snapctl is-connected --list command works.
+
+prepare: |
+  snap pack test-snap
+  snap install --dangerous test-snap_1_all.snap
```

There is a helper that will pack and install the test snap for you:

```suggestion
  "$TESTSTOOLS"/snaps-state install-local test-snap
```